chore: update comparison blog (#10738)
PostHog/posthog.com: contents/blog/posthog-vs-sentry.mdx

## Comparing PostHog and Sentry

> Tracking exceptions within PostHog allows you to easily integrate with all of the other products we offer. You can easily create insights to report errors over time, watch replays of users encountering an exception or target surveys when a user faces a bug.

Reviewer: ```suggestion
Tracking exceptions in PostHog enables you to easily integrate with all of the other products we offer. You can easily create insights to track errors over time, watch replays of users encountering an exception, or target surveys when a user faces a bug.
```

> Given PostHog already tracks context about your users it is possible to understand the impact of exceptions more accurately. For example, you can filter issues affecting VIP user cohorts or find those affecting the most amount of sessions.

Reviewer: ```suggestion
Given PostHog already tracks context about your users, it is possible to understand the impact of exceptions more accurately. For example, you can filter issues affecting VIP user cohorts or find those affecting the most amount of sessions.
```
